ScrumptiousJellyfish/WorkforceManagement
# Ev computers

Please structure your pull requests this way:
- **Purpose of PR**

&nbsp;


- **Issue number this request impacts** ------


&nbsp;

- **Description of what was changed:**

I created the computer controller, and started the create section of the employee controller
&nbsp;

- **Thorough list of steps to test**

1.From the root of the project directory
2. cd into the master branch and run the following command: git fetch --all
3.git checkout ev-computers
4. Open Visual Studio Code and run the debugger by clicking the green button at the top of the screen. 5. The build should run with out any errors and the terminal should pop up
6. Copy and Paste the following url in your Google Chrome browser http://localhost:60041/computer
If all computer data shows up,  then everything in this pull request should've been fulfilled!
&nbsp;


if you don't do this you will be
**FIRED**
